### DISCUSSION TITLE
私钥选择文件功能有一些小问题

### DISCUSSION THREAD

**Discussion by WindyCloudCute** [lollipopkit/flutter_server_box#221] [Ideas]
使用文件选择方式添加保存的时候会提示，但是粘贴进去的却又没有问题。

FormatException: PEM header must end with -----

**Comment by adminpython** [comment]
我也出现这个问题了，直接导入会报错。手动复制粘贴就不会

**Comment by Butwhy1** [comment]
我的无论导入密钥文件还是复制密钥进去都会报这个错误，可能是换行符的问题.
试了几个密钥，最后不要header把密钥复制替换到另一个密钥替换复制居然成功了.
复制到vscode，粘贴进去又报错了，哈哈哈

**Comment by lollipopkit** [comment]
不好意思，之前修复了但是忘回复了，麻烦更新后测试，如果仍有问题，请开启issue
